blackbox/docs/configuration.txt

```diff
@@ -493,6 +493,18 @@ Repositories are used to :ref:`backup <snapshot-restore>` a CrateDB cluster.
 
 See also the :ref:`path.repo <conf-path-repo>` Setting.
 
+Queries
+-------
+
+.. _conf-indices-query-bool.max_clause_count:
+
+**indices.query.bool.max_clause_count**
+  | *Default:* ``8192``
+  | *Runtime:* ``no``
+
+  Specify the maximum number of individual clauses that a boolean query can
```

Comment:
Shouldn't we explain this more in terms of the SQL expressions where this applies to? Afaik a simple `x or x` or `x and x` isn't affected by this setting?

Furthermore, I think it would be good to also explain why this limit is there (e.g. resource consumption / runtime behaviour) if it's increased